File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VoiceFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Voice;

public class VoiceFilterValidator extends FilterValidator<Voice> {

    public VoiceFilterValidator(CustomValidator<Voice> validator) {
        super(Voice.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VoiceFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Voice
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class VoiceFilterValidator(validator: Validator<Voice>) : FilterValidator<Voice>(Voice::class, validator)

